docs/src/main/tut/quickstart.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: docs
+title: Quickstart
+permalink: /docs/rpc/quickstart
+---
+
+# Quickstart
+
+## What’s frees-rpc
+
+[frees-rpc] provides the ability to combine [RPC] protocols, services, and clients in your `Freestyle` program, thanks to [gRPC]. Although it's fully integrated with [gRPC], there are some important differences when defining the protocols, as we’ll see later on, since [frees-rpc] follows the same philosophy as `Freestyle` core, being macro-powered.
+
+## Installation
+
+`frees-rpc` is cross-built for Scala `2.11.x` and `2.12.x`.
+
+It's divided into multiple and different artifacts, grouped by scope:
+
+* `Server`: specifically for the RPC server.
+* `Client`: focused on the RPC auto-derived clients by `frees-rpc`.
+* `Server/Client`: used from other artifacts for both Server and Client.
+* `Test`: useful to test `frees-rpc` applications.
+
+*Artifact Name* | *Scope* | *Mandatory* | *Description*
+--- | --- | --- | ---
+`frees-rpc-server` | Server | Yes | Needed to attach RPC Services and spin-up an RPC Server.
+`frees-rpc-client-core` | Client | Yes | Mandatory to define protocols and auto-derived clients.
+`frees-rpc-client-netty` | Client | Yes* | Optional if you use `OkHttp`, required from the client perspective.
```

Comment:
Not about the document split as such, but is this line accurate? I could see it meaning "netty is optional because if you use okhttp then you don't need to depend on netty" but it seems a bit confusing. I suggest changing it to just "Mandatory on the client side if using Netty". Same goes for the following line.

Reply:
Makes sense to me. Changed description to  "Mandatory on the client side if we are using Netty in the server side"